etna/models/nn/tft_native/tft.py

```diff
@@ -0,0 +1,691 @@
+            Pytorch lightning trainer parameters (api reference :py:class:`pytorch_lightning.trainer.trainer.Trainer`)
+        train_dataloader_params:
+            parameters for train dataloader like sampler for example (api reference :py:class:`torch.utils.data.DataLoader`)
+        test_dataloader_params:
+            parameters for test dataloader
+        val_dataloader_params:
+            parameters for validation dataloader
+        split_params:
+            dictionary with parameters for :py:func:`torch.utils.data.random_split` for train-test splitting
+                * **train_size**: (*float*) value from 0 to 1 - fraction of samples to use for training
+
+                * **generator**: (*Optional[torch.Generator]*) - generator for reproducible train-test splitting
+
+                * **torch_dataset_size**: (*Optional[int]*) - number of samples in dataset, in case of dataset not implementing ``__len__``
+        """
+        self.n_heads = n_heads
+        self.num_layers = num_layers
+        self.dropout = dropout
+        self.hidden_size = hidden_size
+        self.lr = lr
+        self.optimizer_params = optimizer_params
+        self.loss = loss
+        super().__init__(
+            net=TFTNativeNet(
+                encoder_length=encoder_length,
+                decoder_length=decoder_length,
+                n_heads=n_heads,
+                num_layers=num_layers,
+                dropout=dropout,
+                hidden_size=hidden_size,
+                lr=lr,
+                static_categoricals=static_categoricals,
+                static_reals=static_reals,
+                time_varying_categoricals_encoder=time_varying_categoricals_encoder,
+                time_varying_categoricals_decoder=time_varying_categoricals_decoder,
+                time_varying_reals_encoder=time_varying_reals_encoder,
+                time_varying_reals_decoder=time_varying_reals_decoder,
+                categorical_feature_to_id=categorical_feature_to_id,
+                optimizer_params=optimizer_params,
+                loss=QuantileLoss() if loss is None else loss,
+            ),
+            encoder_length=encoder_length,
+            decoder_length=decoder_length,
+            train_batch_size=train_batch_size,
+            test_batch_size=test_batch_size,
+            train_dataloader_params=train_dataloader_params,
+            test_dataloader_params=test_dataloader_params,
+            val_dataloader_params=val_dataloader_params,
+            trainer_params=trainer_params,
+            split_params=split_params,
+        )
+
+    def params_to_tune(self) -> Dict[str, BaseDistribution]:
+        """Get default grid for tuning hyperparameters.
+
+        This grid tunes parameters: ``num_layers``, ``hidden_size``, ``lr``, ``encoder_length``.
+        Other parameters are expected to be set by the user.
+
+        Returns
+        -------
+        :
+            Grid to tune.
+        """
+        return {
```

Comment:
Why did you choose these parameters? In the paper the set was a little bit different